client/templates/userprofile/profile.js

```diff
@@ -6,4 +6,53 @@ Template.profile.helpers({
 		var first = Meteor.user().realname.split(' ');
 		return first[0];
 	}
+});
+
+Template.profile.events({
+	'click .edit-bio': function() {
+		if (Session.get("gearclicked"))
```

Comment:
Also, add comments throughout this function. Each line should have a corresponding plain-language comment.